src/extension.c

```diff
@@ -11,11 +11,11 @@
 #include <utils/lsyscache.h>
 #include <utils/inval.h>
 
-#include "compat-msvc-enter.h" /* To label externs in extension.h and
+#include "compat/compat-msvc-enter.h" /* To label externs in extension.h and
```

Comment:
Why do we have separate enter and exit files? Wouldn't it make more sense to have a single file with MSVC support?

Reply:
certain includes need to be wrapped in this to work on windows